fastlane_core/lib/fastlane_core/analytics/action_completion_context.rb

```diff
@@ -8,10 +8,12 @@ class ActionCompletionStatus
 
   class ActionCompletionContext
     attr_accessor :p_hash
+    attr_accessor :app_id
```

Comment:
Might want to explain in a comment here somewhere that this is not actually sent out to analytics